test-override-system.cjs

```
@@ -0,0 +1,160 @@
+
+// Paths
+const localSopsDir = path.join(__dirname, 'local-sops');
+const originalSopPath = path.join(__dirname, 'src', 'sops', 'ice-cream-test.json');
+const overrideSopPath = path.join(localSopsDir, 'ice-cream-test.json');
+
+console.log('📦 System Paths:');
+console.log(`✅ Original SOP: ${originalSopPath}`);
+console.log(`✅ Override Dir: ${localSopsDir}`);
+console.log(`✅ Override SOP: ${overrideSopPath}\n`);
+
+const fs = require('fs');
```

Biome (2.1.2)

[error] Shouldn't redeclare 'fs'. Consider to delete it or rename it.

'fs' is defined here:


(lint/suspicious/noRedeclare)